packages/app/src/App.tsx

```diff
@@ -12,10 +12,22 @@ import { AsyncStorageExample } from "./AsyncStorageExample";
 import { subplatform } from "./config";
 import LogoSrc from "./logo.png";
 
+// Conditionally import the TV specific react-native types
+// Note: the eval is necessary to avoid static analysis errors
+(async () => {
+  if (Platform.isTV) {
+    eval("import('../../tv/node_modules/react-native/tvos-types.d');");
+  }
+})();
```

Comment:
First time I see this workaround 😮 
Not sure how it would work, but can we move it to the `tv` workspace (maybe in its `index`?) instead of adding platform-specific logic for typings here?

Reply:
Yeah I will not say I'm proud of it but it does the trick 😅.
I tried adding it to the `tv` `index.js` which works but `app` `App.tsx` only knows the normal `react-native` types so it complains. I was trying to think of a better solution the other night but since they are both under the same package name I think our only option is to optionally load the tv one. Thoughts?

Reply:
Actually I just realized the current eval way doesn't work anyway because `app`'s static analysis doesn't recognize that we will load the types so there is a type error  -- I saw `TVViewProps` and thought good to go haha but that's part of `react-native` 😳. Anyway, the import adds types that don't conflict with the existing `react-native` types so it's up to you if you want to just import it in `app` so that all the typings are available or let people import the types if they want. I am going to move the import to `config.js` for now but let me know.

Reply:
I'd just remove them, and then I'll add a bit about it in the README.md 👍 
Once it's removed, we can merge it. Thanks again for looking into that!

Reply:
Cool cool 👍🏽